src/c/tests/zkServer.sh

```diff
@@ -77,7 +77,7 @@ fi
 
 if [ "x${base_dir}" == "x" ]
 then
-zk_base="../../"
+zk_base="../../../"
```

Comment:
is this correct? how did this work before?

Reply:
and is it related to this change? ;)

Reply:
Yes, seems the old code is not working if the base_dir is not specified. 

We were using this to test the change manually, so it's related from testing purpose.